# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../acdctl/ids.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- acdctl-1.1/acdctl.c	2005-08-14 14:46:20.000000000 +0200
+++ acdctl-1.1-fixed/acdctl.c	2007-05-28 17:41:58.892612048 +0200
@@ -80,6 +80,7 @@
 struct display_dev displays[] = {
   { APPLE_VENDOR, 0x9218, "Apple Cinema HD Display 23\"", 2002 },
   { APPLE_VENDOR, 0x921d, "Apple Cinema Display 20\"", 0 },
+  { APPLE_VENDOR, 0x9222, "Apple Cinema Display 20\"", 2006 },
   { 0, 0, "User Specified", 0 },
   { 0, 0, NULL, 0 },
 };
